File: VmsUI/src/app/components/ride-sharing/rides.ts
import { Time } from "@angular/common";

export class Rides {
    ride_ID: number = 0;
    ride_source: string = "";

    ride_destination: string = "";

    depature_time: Time = { hours: 0, minutes: 0 };

    arrival_time: Time = { hours: 0, minutes: 0 };

    ride_duration: number = 0;

    ride_km: number = 0;

    ride_rate: number = 0;
}

File: AdminService/src/main/java/com/admin/model/Rides.java
package com.admin.model;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

import java.sql.Time;
@AllArgsConstructor
@NoArgsConstructor
@Getter
@Setter
public class Rides {
    private int ride_ID;

    private String ride_source;

    private String ride_destination;

    private Time depature_time;

    private Time arrival_time;

    private int ride_duration;

    private float ride_km;

    private float ride_rate;
}
